app/overrides/frontend_payment_add_braintree_dropin.rb

```diff
@@ -0,0 +1,8 @@
+Deface::Override.new(
```

Comment:
We can probably avoid this override by adding an step to the install generator to include this in the frontend js bundle.

See this example from multi_domain:
https://github.com/solidusio/solidus_multi_domain/blob/master/lib/generators/solidus_multi_domain/install/install_generator.rb#L6-L9